.jules/sentinel.md

```
@@ -0,0 +1,4 @@
+## 2026-04-26 - Parameterizing IN Clauses in JPQL
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)